File: iOS/Texas Daily/TexasFact.swift
//
//  TexasFacts.swift
//  Texas Daily
//
//  Created by Ramon Dominguez on 12/3/25.
//
// TexasFact.swift
// TexasFact.swift
// TexasFact.swift
import Foundation

struct TexasFact: Identifiable, Codable {
    let id: Int
    let fact: String
    let category: String
    let date: String?
    let background: String
    let source: String
}

File: Android/Project/app/src/main/java/com/refuge/texasdaily/data/TexasFact.kt
package com.refuge.texasdaily.data

import com.google.gson.annotations.SerializedName

data class TexasFact(
    val id: Int,
    val fact: String,
    val category: String,
    val date: String?,
    val background: String,
    val source: String
)

data class FactsWrapper(
    @SerializedName("facts") val facts: List<TexasFact>
)
